src/v/coproc/tests/pacemaker_tests.cc

```diff
@@ -165,126 +164,3 @@ FIXTURE_TEST(test_copro_auto_deregister_function, coproc_test_fixture) {
           std::move(map), true, std::logical_and<>());
     }).get();
 }
-
-/// Ensures that a deletion of a materialized topic doesn't conflict
-/// with an active coprocessor producing onto it. Furthermore asserts the topic
-/// exists at tests end and contains all expected records.
-FIXTURE_TEST(test_copro_delete_topic, coproc_test_fixture) {
```

Comment:
does this `FIXTURE_TEST_EXPECTED_FAILURES` still work?

Reply:
I do see references to that in the codebase so I assume it would, but this test may sometime pass and sometimes fail, so maybe its not a good use here because this one particular test isn't expected to always fail.